app/scripts/filters/findProject.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .filter('findProject', function(annotationFilter) {
```

Comment:
Can you replace this implementation with one that uses `KeywordService.generateKeywords` and `KeywordService.filterForKeywords`?